libs/vkd3d/memory.c

```diff
@@ -489,9 +489,18 @@ static void vkd3d_memory_transfer_queue_clear_allocation(struct vkd3d_memory_tra
 
     if (allocation->cpu_address)
     {
+        struct d3d12_device *device = queue->device;
+        const struct vkd3d_vk_device_procs *vk_procs = &device->vk_procs;
```

Comment:
Move this line above. Need inverse :christmas_tree: order of declarations.

Reply:
hopefully I've done that right